src/angularjs/src/app/components/filters/neighborhood-filter.directive.js

```diff
@@ -0,0 +1,58 @@
+(function() {
+
+    /**
+     * @ngdoc controller
+     * @name pfb.components.filters.NeighborhoodFilterController
+     *
+     * @description
+     * Controller for the neighborhood filtering table header
+     */
+    /** @ngInject */
+    function NeighborhoodFilterController() {
+        var ctl = this;
+        initialize();
+
+        function initialize() {
+            ctl.filters = {}
+            ctl.nameFilter = '';
+            ctl.stateFilter = '';
+            ctl.countryFilter = '';
+            ctl.onFilterChanged = onFilterChanged;
+        }
+
+        function onFilterChanged() {
+            ctl.filters = {
+                name: ctl.nameFilter,
+                state: ctl.stateFilter,
+                country: ctl.countryFilter
+            };
+        }
+    }
+
+    /**
+     * @ngdoc directive
+     * @scope
+     * @name pfb.components.filters.NeighborhoodFilterDirective:pfbNeighborhoodFilter
+     *
+     * @description
+     * Directive for the neighborhood table header
+     * Filters by neighborhood and status with client-side auto-complete
```

Comment:
Trivial, but this line is the same as the one in `AnalysisJobFilterDirective` and doesn't quite apply here. Could revise to fit, or just drop the line.